Libraries/Opc.Ua.Client/Session.cs

```diff
@@ -174,30 +175,20 @@ public Session(ITransportChannel channel, Session template, bool copyEventHandle
 
             if (m_endpoint.Description.SecurityPolicyUri != SecurityPolicies.None)
```

Comment:
try to move the async portions out of the constructor, then additional calls have to be made to load the certs